applications/luci-app-tor-hs/htdocs/luci-static/resources/view/tor_hs_status.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+'require view';
+'require dom';
+'require ui';
+'require rpc';
+'require form';
+
+var callTorHSList = rpc.declare({
+		object: 'tor-hs-rpc',
+		method: 'list-hs',
+		expect: {'':{}  }});
```

Comment:
The indentation does not fit. only one tab and the parenthesis at the end of a new line. Please look at other modules how they do this